concept/thing/impl/ThingImpl.java

```diff
@@ -297,16 +298,16 @@ public final boolean isRemote() {
             return rpcTransaction;
         }
 
-        Stream<ThingImpl> stream(ConceptProto.Thing.Req.Builder method, Function<ConceptProto.Thing.Res, ConceptProto.Thing> thingGetter) {
+        Stream<ThingImpl> stream(ConceptProto.Thing.Req.Builder method, Function<ConceptProto.Thing.Res, List<ConceptProto.Thing>> thingGetter) {
```

Comment:
Here each `res` may contain a list of answers, so we change the signature (and all similar functions) to accept to transform `res` to a list of items instead of just one.